contributor/Member_Application.md

```diff
@@ -0,0 +1,8 @@
+# Member Application
+To be a member of OpenLab follow the simple steps below:
+
+1. Open an issue to [apply to join](https://github.com/theopenlab/openlab/issues/new?template=joinin.md) first, fill out the form completely with as much detail, for example your contact information and the way you will support to OpenLab
+2. The governance team owner will assign a role to you based on your information, then will contact you and send you an invitation email, please confirm it and reply to us
+3. Then you are the member of OpenLab, please visit the OpenLab project [status board](https://github.com/orgs/theopenlab/projects/1) to get the overview of what's going on
+4. The application issue will be closed by governance team owner
+5. Welcome to the playground! 
```

Comment:
4. Welcome to the playground! Say "Hi" in OpenLab IRC channel `#askopenlab` on freenode.net :-)
5. The application issue will be closed by governance team owner